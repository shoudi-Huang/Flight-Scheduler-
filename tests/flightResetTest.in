location import locations.csv
flight add sunday 20:00 Berlin NewYork 250
flight 0
flight gh reset
flight 1 reset
flight 0 reset
flight 0
flight 0 book
flight 0
FLIGHT 0 RESET
flight 0
flight 0 book 250
flight 0
fLiGht 0 ResEt 0 extra bit
flight 0
exit
