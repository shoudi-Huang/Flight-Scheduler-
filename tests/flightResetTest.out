User: Imported 51 locations.

User: Successfully added Flight 0.

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1825.24
Passengers:   0/250

User: Invalid Flight ID.

User: Invalid Flight ID.

User: Reset passengers booked to 0 for Flight 0, Sun 20:00 Berlin --> NewYork.

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1825.24
Passengers:   0/250

User: Booked 1 passenger on flight 0 for a total cost of $1825.24

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1822.32
Passengers:   1/250

User: Reset passengers booked to 0 for Flight 0, Sun 20:00 Berlin --> NewYork.

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1825.24
Passengers:   0/250

User: Booked 250 passengers on flight 0 for a total cost of $433914.59
Flight is now full.

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1988.58
Passengers:   250/250

User: Reset passengers booked to 0 for Flight 0, Sun 20:00 Berlin --> NewYork.

User: Flight 0
Departure:    Sun 20:00 Berlin
Arrival:      Mon 04:52 NewYork
Distance:     6,386km
Duration:     8h 52m
Ticket Cost:  $1825.24
Passengers:   0/250

User: Application closed.
